base_layer/transactions/src/aggregated_body.rs

```diff
@@ -84,8 +83,7 @@ impl AggregateBody {
         &self.kernels
     }
 
-    /// Available for tests only. Get a mutable reference to the inputs
-    #[cfg(test)]
```

Comment:
This is needed in tari_core tests

Reply:
hmmm, ja this is not ideal. But this whole split will need to be revisited I think.